### ISSUE TITLE
Emergency message is sent to wall while restarting haproxy-admin

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5242]
**Describe the bug**
When I restart haproxy-admin, it throws an emergency message on the wall:
```
Message from syslogd@pf-julien at Mar 26 08:27:21 ...-julien:/usr/local/fingerbank- 2:root@pf-julien:~/src/gospace/src/git.inverse.ca/inverse/fingerbank-collector  3:root@pf-julien:/usr/local/pf  4:root@pf-julien:~/bpftools/linux_tools                                                                                        "printf "\033k%s@%s:%s\" 08:27 26-Mar-20
 haproxy[17919]:backend 172.20.20.109-admin has no server available!
```

This is not necessary

**Comment by fdurand** [comment]
It's because we have the old admin backend in the config.
I will check if it's possible to disable.

**Comment by fdurand** [comment]
Found in the haproxy forum:

```
This message will be sent to stderr if one of the following is true:

haproxy is currently starting or
haproxy is configured with the verbose option or
haproxy is not configured with the quiet option
```

Since it's start then we have the message.

**Comment by fdurand** [comment]
Fixed by 9ca04f270fcb5e442e16720c1f949a593dabbb1b
